.changeset/dirty-guests-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/wallets": minor
+---
+
+feat(wallets): add onAuthSuccess callback for paperWallet
```

Comment:
nit: you don't need the `feat(...)` here because it's already "scoped" to the package in the changeset itself